dd-trace-py/Dockerfile

```diff
@@ -1,14 +1,13 @@
 # Latest image for this Dockerfile: datadog/docker-library:ddtrace_py
 
-# DEV: Use `python:slim` instead of an `alpine` image to support installing wheels from PyPI
+# DEV: Use `debian:slim` instead of an `alpine` image to support installing wheels from PyPI
```

Comment:
```suggestion
# DEV: Use `debian:stretch-slim` instead of an `alpine` image to support installing wheels from PyPI
```

?

Reply:
Yeah I did not mention the version so it did not have to be out of date when we upgrade ;)

Reply:
works for me